apiserver/facades/client/modelmanager/modelmanager.go

```diff
@@ -49,6 +50,14 @@ var (
 	supportedFeaturesGetter = stateenvirons.SupportedFeatures
 )
 
+// ModelManagerV10 defines the methods on the version 10 facade for the
+// modelmanager API endpoint.
+type ModelManagerV10 interface {
+	ModelManagerV8
```

Comment:
We normally would embed V9

Reply:
We have to embed 8 here. Because we dropped the changes in v9 and revert to use v8 as the base.